src/Testcontainers.LocalStack/AwsService.cs

```diff
@@ -0,0 +1,132 @@
+﻿namespace Testcontainers.Minio;
```

Comment:
```suggestion
﻿namespace Testcontainers.LocalStack;
```

It looks like the namespace is wrong in all files. Please move the usings to global usings too.

Reply:
fact, fixed it